templates/zombienet/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "template-zombienet-tests"
+description = "Zombienet test for templates."
+version = "0.0.0"
+license = "Unlicense"
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+edition.workspace = true
+publish = false
+
+[dependencies]
+env_logger = "0.11.2"
+log = "0.4"
+tokio = { version = "1.36.0", features = ["rt-multi-thread"] }
+anyhow = "1.0.81"
+zombienet-sdk = "0.2.8"
+
+[features]
+zombienet = []
```

Comment:
Again, this is not needed, as this crate is no longer part of `templates/parachain`. If you run `cargo test` in `templates/parachain`, this crate is not called.

Reply:
Thanks for the feedback @kianenigma! The idea behind the feature flag is to prevent failures in these tests for users that run `cargo test --workspace` in the polkadot-sdk repo, since this tests need some extra steps (build the binaries) in order to run. So, I think we can keep the feature and set in our CI to ensure block production for the temaplates.

Thx!